bin/copyProjectDataToPrm.sh

```diff
@@ -280,16 +280,14 @@ function rsyncProjectRun() {
 	#
 	if [[ -e "${JOB_CONTROLE_FILE_BASE}.failed" ]]; then
 		log4Bash 'ERROR' "${LINENO}" "${FUNCNAME:-main}" '0' "Found ${JOB_CONTROLE_FILE_BASE}.failed. Setting track & trace state to failed :(."
-		trackAndTracePut 'status_projects' "${_project}" 'copy_results_prm' 'failed'
 
 	elif [[ -e "${JOB_CONTROLE_FILE_BASE}.finished" ]]; then
 		echo "Project/run ${_project}/${_run} is ready. The data is available at ${PRM_ROOT_DIR}/projects/." \
 			>> "${JOB_CONTROLE_FILE_BASE}.finished"
 		log4Bash 'INFO' "${LINENO}" "${FUNCNAME:-main}" '0' "Found ${JOB_CONTROLE_FILE_BASE}.finished. Setting track & trace state to finished :)."
 
-		trackAndTracePut 'status_projects' "${_project}" 'copy_results_prm' 'finished'
 		dateFinished=$(date +%FT%T%z -r "${JOB_CONTROLE_FILE_BASE}.finished")
-		trackAndTracePut 'status_projects' "${_project}" 'finishedDate' "'${dateFinished}'"
+		echo "\"${dateFinished}\"" > "${JOB_CONTROLE_FILE_BASE}.trace_putFromFile_projects.csv"
```

Comment:
Werkt wel, maar als de formatting wat complexer wordt zoals hier met extra letterlijke quotes, is het handiger om de formatting en de waardes van elkaar de scheiden met `printf`:
```
printf '"%s"\n' "${dateFinished}" > "${JOB_CONTROLE_FILE_BASE}.trace_putFromFile_projects.csv"
```